experiments/AtmosGCM/moist-baroclinic-wave-bulksfcflux.jl

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env julia --project
+
+using ArgParse
+using LinearAlgebra
+using StaticArrays
+using Test
+
+using ClimateMachine
+using ClimateMachine.Atmos
+using ClimateMachine.Orientations
+using ClimateMachine.ConfigTypes
+using ClimateMachine.Diagnostics
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.SystemSolvers: ManyColumnLU
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.TemperatureProfiles
+using ClimateMachine.Thermodynamics:
+    air_density, air_temperature, total_energy, internal_energy, PhasePartition, air_pressure
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.VariableTemplates
+
+using CLIMAParameters
+using CLIMAParameters.Planet:
+    day, grav, R_d, Omega, planet_radius, MSLP, T_triple, press_triple, LH_v0, R_v
+using CLIMAParameters.Atmos.SubgridScale:
+    C_drag
+
+struct EarthParameterSet <: AbstractEarthParameterSet end
+#press_triple(::EarthParameterSet) = 610.78      # from Thatcher and Jablonowski (2016)
+#C_drag(::EarthParameterSet) = 0.0044            # "
+
+# driver-specific parameters added here
+T_sfc_pole(::EarthParameterSet) = 271.0
```

Comment:
Should this be added to CLIMAParameters?

Reply:
This is specific to the Thatcher&Jablonowski experiment, so I wouldn't have thought so. :)